app/views/idv/vendor_outage/show.html.erb

```diff
@@ -0,0 +1,63 @@
+<%= render StepIndicatorComponent.new(
+      steps: step_indicator_steps,
+      current_step: :getting_started,
+      locale_scope: 'idv',
+      class: 'margin-x-neg-2 margin-top-neg-4 tablet:margin-x-neg-6 tablet:margin-top-neg-4',
+    ) %>
+
+<%= render(AlertIconComponent.new) %>
```

Comment:
This looks like it might fit the format of `StatusPageComponent` ?

Reply:
Done in [44625c1](https://github.com/18F/identity-idp/pull/8011/commits/44625c1b996b98e461e410fdddf5ff71e8322e91)